bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -136,40 +136,59 @@ export class AxisView extends GuideRendererView {
     this._draw_oriented_labels(ctx, labels, coords, orient, this.panel.side, standoff, visuals)
   }
 
+  protected _axis_label_extent(): number {
+    const text = this.model.axis_label
+    if (!text)
+      return 0
+
+    const axis_label = new TextBox({text})
+    axis_label.angle = this.panel.get_label_angle_heuristic("parallel")
+    axis_label.visuals = this.visuals.axis_label_text
+
+    const size = axis_label.size()
+    const extent = this.dimension == 0 ? size.height : size.width
+
+    const standoff = this.model.axis_label_standoff
+    const padding = 3
```

Comment:
@bryevdv, this is what I mentioned on Wednesday. Previously padding depended on label width (10%), font descent/ascender and orientation (depending on axis configuration). Now the intrinsic padding is 0, so I'm adding 3px here to make it look better. The question is what should be the value, do we want to make this configurable?

Reply:
Well, I don't know what the value should be offhand, so my default would be to make it configurable just in case whatever we do choose is not the best in some situtations.